### DISCUSSION TITLE
Would a dedicated VIM comparison be appropriate?

### DISCUSSION THREAD

**Discussion by ideasman42** [meow-edit/meow#233] [Ideas]
While it's not necessary to define Meow by it's differences to VIM (evil-mode), I think it would be useful to have a document that shows examples of actions in VIM, with the equivalent key bindings in Meow.

With 10 or more of these I think it would help VIM users quickly map typical workflow to Meow and help them get an idea of the kinds of differences they should expect with Meow.

**Comment by eshrh** [comment]
This is a good idea in my opinion. It could either go into the tutorial document or it could be a new document as well (not sure if there's enough content).

Would you be interested in writing this?

**Reply by snowiow** [reply]
Would also be happy to contribute. Used VIM for many years now and I'm currently trying to find my way around in meow.

**Reply by DogLooksGood** [reply]
How about making a short video?

**Reply by ideasman42** [reply]
Prefer text, much easier to update and expand on.

Also, I think this kind of document lends it's self to a table with two columns: `VIM | Meow` showing actions in both.

**Comment by snowiow** [comment]
I started with a written comparison [here](https://github.com/meow-edit/meow/pull/238). Feel free to add to it and edit existing things if they don't have the most effective equivalent in meow. I would be especially glad if the current blanks will be filled out, because those I still don't have find an alternative for in meow.

Also I didn't see the suggestion for the table approach so far, instead it's a categorized list with `vim binding (explanation what it does): meow command`, I could change this as well if it's wanted.

**Reply by DogLooksGood** [reply]
Good job! I also added some information to the PR.
